function restore_exception_handler () {
    
}